### ISSUE TITLE
Transfer learning to refine predicted library

### ISSUE THREAD

**Issue by snsipe** [MannLabs/alphapeptdeep#145]
Hello,
I am learning to refine our predicted spectral library to better match various samples (various proteolytic peptides, novel modifications, etc.). I am starting with standard unmodified tryptic peptides to ensure I am using AlphPeptDeep correctly, but I have run into issues with the file uploads. I have tried importing various speclib_tsv libraries (initially with then without modifications) as well as diann reports but am running into the same errors. I did not locate any log output, but attached is the output from my Windows PowerShell
[2024-02-22_log.txt](https://github.com/MannLabs/alphapeptdeep/files/14377364/2024-02-22_log.txt)

I suspect I am loading inappropriate/invalid files. Are there any example file formats that I can test with?

**Comment by jalew188** [comment]
Could you please check diann report file? "Modified.Sequence" must be in the header

**Comment by snsipe** [comment]
Yes, "Modified.Sequence" is one of the column headers for each of the reports I have tried.
Would you be able to provide an example report and raw file that I may try through the gui? Perhaps there are other setting that I am not considering, e.g. modificaitons.

**Comment by yanzhou-boy** [comment]
The file path you upload contains multiple files, and the software cannot read the corresponding files. You only need to delete the remaining files and keep only one psm file
